src/DependenciesPackager/Program.cs

```diff
@@ -184,7 +189,7 @@ static int Main(string[] args)
                 packagesVersion,
                 useCli,
                 quiet,
-                runtimes,
+                runtime,
```

Comment:
Then this should be architecture instead of runtime. You can't crossgen linux bits on windows. I doubt we can crossgen x64 bits on x86.